app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java

```diff
@@ -364,9 +377,17 @@ public void saveAlert(final models.ebean.Alert alert) {
          * {@inheritDoc}
          */
         @Override
-        public long getEtag() {
-            final SqlQuery sqlQuery = Ebean.createSqlQuery("SELECT CURRVAL('portal.alerts_etag_seq') AS etag;");
-            return sqlQuery.findUnique().getLong("etag");
+        public long getEtag(final Organization organization) {
+            final SqlQuery sqlQuery = Ebean.createSqlQuery("SELECT etag from portal.alerts_etags where organization = ?;");
```

Comment:
The thing we're representing is an etag.  Etags are not generic versions.  They don't need to increment to be valid (though they do in this case).  I'm not opposed to calling it something else in the database, but it's not just a version.  And as you pointed out, it's a "version" on a view of the alerts.  It's just the current implementation that we only version at the organization level.  For this reason, I'm strongly opposed to putting it in the organization table.  It's not an attribute of an organization.  It's an implementation detail about the alerts or expressions query.